__tests__/app_testing.test.js

```diff
@@ -27,3 +27,14 @@ describe("api/topics", () => {
       });
   });
 });
+
+describe("/api", () => {
+  test("GET:200 Should return object containing all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ text }) => {
+        expect(typeof JSON.parse(text)).toEqual("object");
```

Comment:
we want to expect to expect the value here rather than just an object. We could have that the text was equal to the endpoing file being brought into this test file :)